app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -0,0 +1,28 @@
+# Thank you for completing your teacher training application
+
+You have submitted an application to:
+
+You can track the progress of your application here:
+https://apply-for-teacher-training.service.gov.uk/applications
+
+Your application reference is <%= @application_ref %>
+
+# Next steps
+
+# References
```

Comment:
Should this be a h2?

Reply:
@tijmenb yes, but how? 🤔 (it is not markdown)
I asked this on slack: https://ukgovernmentdfe.slack.com/archives/CN1MCQCHZ/p1571661730010500

Reply:
"Notify flavoured markdown" does not support more than one heading type. I think this should be **bold** instead :)